EVGEN/AliGenPerformance.cxx

```diff
@@ -88,35 +90,35 @@ AliGenPerformance::AliGenPerformance():
 }
 //-----------------------------------------------------------------------------
 AliGenPerformance::AliGenPerformance(const char* generName, Int_t verboseLevel):
-  AliGenerator(),
-  fNJets(1),                // mean number of jets per event
-  fF1Momentum(0),           // momentum distribution function 
-  fFPhi(0),                 // phi distribution function
-  fFTheta(0),               // theta distribution function
-  fFPosition(0),            // position distribution function 
-  fFPdg(0),                 // pdg distribution function  
-  fStreamer(0),           // test stream - used for tuning of parameters of generator
-  fVerboseLevel(0)            // verbose level
+        AliGenerator(),
+        fNJets(1),                // mean number of jets per event
+        fF1Momentum(0),           // momentum distribution function
+        fFPhi(0),                 // phi distribution function
+        fFTheta(0),               // theta distribution function
+        fFPosition(0),            // position distribution function
+        fFPdg(0),                 // pdg distribution function
+        fStreamer(0),           // test stream - used for tuning of parameters of generator
+        fVerboseLevel(0)            // verbose level
```

Comment:
@sawenzel 
We discussed with Dario re-formatting issue some time ago but we did not conclude.

What I did in this commit:
I recently started to use CLIon for editing and debugging. 
Unfortunately old default indentation of xemacs is not compatible with CLion indentation  - and code was difficult to read. I applied CLion automatic indentation. 

I would like to ask following question:
* Can we define default AliRoot formatting?
  * I would like to apply this formatting in CLIon and xemacs
*   Should we reformat class by class, before actual modifying of the code?

I prefer to conclude as soon as in the next code modification I will have again the same problem.

Marian